chore(components): Update README and reduce output size of KFServing component (#5386)
kubeflow/pipelines: components/kubeflow/kfserving/README.md

# KFServing Component

This is the Kubeflow Pipelines component for KFServing. This uses the [V1beta1 API](https://github.com/kubeflow/kfserving/blob/master/docs/apis/v1beta1/README.md),
so your cluster must have a KFServing version >= v0.5.0 in order to use this.

> For those still using an older version of KFServing less than v0.5.0, an older version of the KFServing Pipelines component must be used

Reviewer: ```suggestion
If you are using KFServing version prior to v0.5.0, an older deprecated version of the KFServing Pipelines component must be used
```
